src/views/Accessibility/index.test.tsx

```diff
@@ -43,7 +43,7 @@ describe('Accessibility wrapper', () => {
     expect(wrapper.length).toBe(1);
   });
 
-  it('renders the "Report a problem" link area component', async () => {
+  it.skip('renders the "Report a problem" link area component', async () => {
```

Comment:
Given that we already have tests for `<HelpFooter>`, do you think we can just get rid of these tests instead of `.skip`ing them? Or maybe we'll just do that in the cleanup ticket?

Reply:
I was planning to remove these tests once `helpFooter` was done with and removed as part of that cleanup ticket